app/views/user_mailer/claim_notification.html.erb

```diff
@@ -25,7 +25,7 @@
     <% end %>
   </ul>
 
-  <p><%= t(".redirects") %></p>
+  <p><%= t(".redirects_html") %></p>
```

Comment:
Would be nice to have the locale keys be `redirects.html` and `redirects.text`, see https://github.com/otwcode/otwarchive/pull/4519#discussion_r1212423492.

Reply:
Note: This is a nitpick. If it's urgent, the rest of the commit looks good and this could be merged as is.